setup.py

```diff
@@ -17,10 +17,9 @@
     python_requires=">=3.6",
     include_package_data=True,
     install_requires=[
-        "fire~=0.5.0",
+        "fire~=0.6.0",

```

Comment:
Hey, I tried to run your fork but I keep getting an error about satisifying `fire`:
```python
ERROR: Could not find a version that satisfies the requirement fire~=0.6.0 (from aitextgen) (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.2.0, 0.2.1, 0.3.0, 0.3.1, 0.4.0, 0.5.0)
```

Reply:
yes you ran into the other part of the medallion.
You either have too loose requirements and it results in this repository not working with newer versions
or you restrict too much and people with different setups cannot use your project.

I set the requirements of this project to be minor-dependent following my current setup.
Your problems describes, that you have a version of fire installed, that is not `0.6.x`. I haven't tested this fork with other version, but I am quite sure you fire can be downgraded and still work.

You can either install the required versions from `requirements.txt`
or change `setup.py` to allow your currently installed version.

I personally think, rather restrict more than less. That is why I changed the requirements to be minor-exact.

Reply:
My apologies. Apparently I was sleeping during my latest push. I set the requirement to be 0.5 for obvious reasons